README.md

```diff
@@ -1,5 +1,19 @@
 # Material Yew
 
+[![Crates.io][crates-badge]][crates-url]
+[![API Docs][docrs-badge]][docrs-url]
+[![Apache-2.0 licensed][license-badge]][license-url]
+[![Website][website-badge]][website-url]
+
+[docrs-badge]: https://img.shields.io/docsrs/material-yew
+[docrs-url]: https://docs.rs/material-yew/
+[crates-badge]: https://img.shields.io/crates/v/material-yew
+[crates-url]: https://crates.io/crates/material-yew
+[license-badge]: https://img.shields.io/github/license/hamza1311/material-yew
+[license-url]: https://github.com/hamza1311/material-yew/blob/master/LICENSE.md
+[website-badge]: https://img.shields.io/website?url=https%3A%2F%2Fmaterial-yew.rm.rs%2F
+[website-url]: https://material-yew.rm.rs/
```

Comment:
Can you inline the link? It's not a big deal but I prefer it that way.

Reply:
Updated, let me know what you think